.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "golang.go",
+    "editorconfig.editorconfig",
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "voorjaar.windicss-intellisense",
+    "lokalise.i18n-ally",
```

Comment:
what is this used for?

Reply:
I will remove it as we haven't added i18n yet (#258). The plugin would resolve translation strings in html later, so you could directly see what the html translation would render to.

Reply:
```suggestion
```